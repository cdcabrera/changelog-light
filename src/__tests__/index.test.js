const { commitChangelog } = require('../index');

describe('Commit Changelog', () => {
  it('should parse commits to produce a CHANGELOG.md', () => {
    expect(commitChangelog({ date: new Date('2022-10-01').toISOString() })).toMatchSnapshot('Commit Changelog');
  });
});
