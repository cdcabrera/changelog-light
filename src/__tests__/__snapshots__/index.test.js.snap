// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commit Changelog should parse commits to produce a CHANGELOG.md: Commit Changelog 1`] = `
{
  "changelog": "# Changelog
All notable changes to this project will be documented in this file.
## <semverClean>["<semverInc>[\\"0.0.100\\",\\"patch\\"]</semverInc>"]</semverClean> (2022-10-01)

",
  "package": "Version bump: patch",
  "parsedCommits": {},
  "semverBump": "patch",
  "semverWeight": 0,
  "version": "<semverInc>["0.0.100","patch"]</semverInc>",
  "versionClean": "<semverClean>["<semverInc>[\\"0.0.100\\",\\"patch\\"]</semverInc>"]</semverClean>",
}
`;
